Include path problem on simple example

Following the example given in the README fails to compile with the error "No such file or directory" on a header include. However, the warnings given earlier show that the header file was in fact found.

build.rs:

```rust
fn main() {
    let path = std::path::PathBuf::from("/Users/psmit/Code/kaldi/tools/openfst/include");
    let mut b = autocxx_build::Builder::new("src/main.rs", &[&path]).extra_clang_args(&["-Wno-unused-parameter","-Wno-sign-compare"]).expect_build();
    // This assumes all your C++ bindings are in main.rs
    b.flag_if_supported("-std=c++14").compile("fst-demo");
    println!("cargo:rerun-if-changed=src/main.rs");
    // Add instructions to link to any C++ libraries you need.
    println!("cargo:rustc-link-search=/Users/psmit/Code/kaldi/tools/openfst/lib");
    println!("cargo:rustc-link-lib=fst");
}

```

main.rs:

```rust
use autocxx::prelude::*;

autocxx::include_cpp! {
    #include "fst/arc.h"
    generate!("fst::StdArc")
    safety!(unsafe)
}

fn main() {
    println!("Hello, world!");
}
```

Result:

```
 cargo build
warning: In file included from /Users/psmit/kaldi-cxx/target/debug/build/kaldi-cxx-b3443cbce5d29980/out/autocxx-build-dir/cxx/gen0.cxx:1:
warning: In file included from /Users/psmit/Code/kaldi/tools/openfst/include/fst/arc.h:15:
warning: In file included from /Users/psmit/Code/kaldi/tools/openfst/include/fst/expectation-weight.h:24:
warning: In file included from /Users/psmit/Code/kaldi/tools/openfst/include/fst/pair-weight.h:18:
warning: In file included from /Users/psmit/Code/kaldi/tools/openfst/include/fst/weight.h:16:
warning: /Users/psmit/Code/kaldi/tools/openfst/include/fst/compat.h:87:23: warning: comparison of integers of different signs: 'int' and 'std::basic_string<char>::size_type' (aka 'unsigned long') [-Wsign-compare]
warning:     for (int i = 0; i < data.size(); ++i) {
warning:                     ~ ^ ~~~~~~~~~~~
warning: In file included from /Users/psmit/kaldi-cxx/target/debug/build/kaldi-cxx-b3443cbce5d29980/out/autocxx-build-dir/cxx/gen0.cxx:1:
warning: In file included from /Users/psmit/Code/kaldi/tools/openfst/include/fst/arc.h:15:
warning: In file included from /Users/psmit/Code/kaldi/tools/openfst/include/fst/expectation-weight.h:24:
warning: In file included from /Users/psmit/Code/kaldi/tools/openfst/include/fst/pair-weight.h:18:
warning: In file included from /Users/psmit/Code/kaldi/tools/openfst/include/fst/weight.h:19:
warning: /Users/psmit/Code/kaldi/tools/openfst/include/fst/util.h:125:62: warning: unused parameter 'v' [-Wunused-parameter]
warning:   return internal::ReadContainerType(strm, c, [](decltype(c) v, int n) {});
warning:                                                              ^
warning: /Users/psmit/Code/kaldi/tools/openfst/include/fst/util.h:125:69: warning: unused parameter 'n' [-Wunused-parameter]
warning:   return internal::ReadContainerType(strm, c, [](decltype(c) v, int n) {});
warning:                                                                     ^
warning: /Users/psmit/Code/kaldi/tools/openfst/include/fst/util.h:130:62: warning: unused parameter 'v' [-Wunused-parameter]
warning:   return internal::ReadContainerType(strm, c, [](decltype(c) v, int n) {});
warning:                                                              ^
warning: /Users/psmit/Code/kaldi/tools/openfst/include/fst/util.h:130:69: warning: unused parameter 'n' [-Wunused-parameter]
warning:   return internal::ReadContainerType(strm, c, [](decltype(c) v, int n) {});
warning:                                                                     ^
warning: /Users/psmit/Code/kaldi/tools/openfst/include/fst/util.h:135:62: warning: unused parameter 'v' [-Wunused-parameter]
warning:   return internal::ReadContainerType(strm, c, [](decltype(c) v, int n) {});
warning:                                                              ^
warning: /Users/psmit/Code/kaldi/tools/openfst/include/fst/util.h:135:69: warning: unused parameter 'n' [-Wunused-parameter]
warning:   return internal::ReadContainerType(strm, c, [](decltype(c) v, int n) {});
warning:                                                                     ^
warning: In file included from /Users/psmit/kaldi-cxx/target/debug/build/kaldi-cxx-b3443cbce5d29980/out/autocxx-build-dir/cxx/gen0.cxx:1:
warning: In file included from /Users/psmit/Code/kaldi/tools/openfst/include/fst/arc.h:15:
warning: In file included from /Users/psmit/Code/kaldi/tools/openfst/include/fst/expectation-weight.h:24:
warning: In file included from /Users/psmit/Code/kaldi/tools/openfst/include/fst/pair-weight.h:18:
warning: /Users/psmit/Code/kaldi/tools/openfst/include/fst/weight.h:232:20: warning: unused parameter 'w1' [-Wunused-parameter]
warning:   W2 operator()(W1 w1) const {
warning:                    ^
warning: In file included from /Users/psmit/kaldi-cxx/target/debug/build/kaldi-cxx-b3443cbce5d29980/out/autocxx-build-dir/cxx/gen0.cxx:1:
warning: In file included from /Users/psmit/Code/kaldi/tools/openfst/include/fst/arc.h:15:
warning: /Users/psmit/Code/kaldi/tools/openfst/include/fst/expectation-weight.h:78:37: warning: unused parameter 'delta' [-Wunused-parameter]
warning:   PairWeight<X1, X2> Quantize(float delta = kDelta) const {
warning:                                     ^
warning: /Users/psmit/Code/kaldi/tools/openfst/include/fst/expectation-weight.h:110:74: warning: unused parameter 'w1' [-Wunused-parameter]
warning: inline ExpectationWeight<X1, X2> Divide(const ExpectationWeight<X1, X2> &w1,
warning:                                                                          ^
warning: /Users/psmit/Code/kaldi/tools/openfst/include/fst/expectation-weight.h:111:74: warning: unused parameter 'w2' [-Wunused-parameter]
warning:                                         const ExpectationWeight<X1, X2> &w2,
warning:                                                                          ^
warning: /Users/psmit/Code/kaldi/tools/openfst/include/fst/expectation-weight.h:112:52: warning: unused parameter 'typ' [-Wunused-parameter]
warning:                                         DivideType typ = DIVIDE_ANY) {
warning:                                                    ^
warning: In file included from /Users/psmit/kaldi-cxx/target/debug/build/kaldi-cxx-b3443cbce5d29980/out/autocxx-build-dir/cxx/gen0.cxx:1:
warning: In file included from /Users/psmit/Code/kaldi/tools/openfst/include/fst/arc.h:16:
warning: /Users/psmit/Code/kaldi/tools/openfst/include/fst/float-weight.h:303:50: warning: unused parameter 'typ' [-Wunused-parameter]
warning:                                       DivideType typ = DIVIDE_ANY) {
warning:                                                  ^
warning: /Users/psmit/Code/kaldi/tools/openfst/include/fst/float-weight.h:538:45: warning: unused parameter 'typ' [-Wunused-parameter]
warning:                                  DivideType typ = DIVIDE_ANY) {
warning:                                             ^
warning: /Users/psmit/Code/kaldi/tools/openfst/include/fst/float-weight.h:709:48: warning: unused parameter 'typ' [-Wunused-parameter]
warning:                                     DivideType typ = DIVIDE_ANY) {
warning:                                                ^
warning: In file included from /Users/psmit/kaldi-cxx/target/debug/build/kaldi-cxx-b3443cbce5d29980/out/autocxx-build-dir/cxx/gen0.cxx:1:
warning: In file included from /Users/psmit/Code/kaldi/tools/openfst/include/fst/arc.h:17:
warning: /Users/psmit/Code/kaldi/tools/openfst/include/fst/lexicographic-weight.h:91:38: warning: unused parameter 'delta' [-Wunused-parameter]
warning:   LexicographicWeight Quantize(float delta = kDelta) const {
warning:                                      ^
warning: In file included from /Users/psmit/kaldi-cxx/target/debug/build/kaldi-cxx-b3443cbce5d29980/out/autocxx-build-dir/cxx/gen0.cxx:1:
warning: In file included from /Users/psmit/Code/kaldi/tools/openfst/include/fst/arc.h:20:
warning: /Users/psmit/Code/kaldi/tools/openfst/include/fst/signed-log-weight.h:157:48: warning: unused parameter 'typ' [-Wunused-parameter]
warning:                                     DivideType typ = DIVIDE_ANY) {
warning:                                                ^
warning: In file included from /Users/psmit/kaldi-cxx/target/debug/build/kaldi-cxx-b3443cbce5d29980/out/autocxx-build-dir/cxx/gen0.cxx:1:
warning: In file included from /Users/psmit/Code/kaldi/tools/openfst/include/fst/arc.h:21:
warning: /Users/psmit/Code/kaldi/tools/openfst/include/fst/sparse-power-weight.h:109:51: warning: unused parameter 'k' [-Wunused-parameter]
warning:   return SparsePowerWeightMap(w1, w2, [](const K &k, const W &v1, const W &v2) {
warning:                                                   ^
warning: /Users/psmit/Code/kaldi/tools/openfst/include/fst/sparse-power-weight.h:118:51: warning: unused parameter 'k' [-Wunused-parameter]
warning:   return SparsePowerWeightMap(w1, w2, [](const K &k, const W &v1, const W &v2) {
warning:                                                   ^
warning: /Users/psmit/Code/kaldi/tools/openfst/include/fst/sparse-power-weight.h:129:47: warning: unused parameter 'k' [-Wunused-parameter]
warning:                               [type](const K &k, const W &v1, const W &v2) {
warning:                                               ^
warning: /Users/psmit/Code/kaldi/tools/openfst/include/fst/sparse-power-weight.h:151:32: warning: unused parameter 'k' [-Wunused-parameter]
warning:       w1, w2, [delta](const K &k, const W &v1, const W &v2) {
warning:                                ^
warning: In file included from /Users/psmit/kaldi-cxx/target/debug/build/kaldi-cxx-b3443cbce5d29980/out/autocxx-build-dir/cxx/gen0.cxx:1:
warning: In file included from /Users/psmit/Code/kaldi/tools/openfst/include/fst/arc.h:22:
warning: /Users/psmit/Code/kaldi/tools/openfst/include/fst/string-weight.h:95:31: warning: unused parameter 'delta' [-Wunused-parameter]
warning:   StringWeight Quantize(float delta = kDelta) const { return *this; }
warning:                               ^
warning: /Users/psmit/Code/kaldi/tools/openfst/include/fst/string-weight.h:288:31: warning: unused parameter 'delta' [-Wunused-parameter]
warning:                         float delta = kDelta) {
warning:                               ^
warning: 23 warnings generated.
   Compiling kaldi-cxx v0.1.0 (/Users/psmit/kaldi-cxx)
error: couldn't read /Users/psmit/kaldi-cxx/target/debug/build/kaldi-cxx-b3443cbce5d29980/out/autocxx-build-dir/rs/fst/arc.h: No such file or directory (os error 2)
 --> /Users/psmit/kaldi-cxx/target/debug/build/kaldi-cxx-b3443cbce5d29980/out/autocxx-build-dir/rs/autocxx-ffi-default-gen.rs:1:1410
  |
1 | ...) -> UniquePtr < CxxString > ; type AutocxxConcrete2 ; include ! ("fst/arc.h") ; include ! ("autocxxgen_ffi.h") ; } extern "Rust" { } ...
  |                                                           ^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the macro `include` (in Nightly builds, run with -Z macro-backtrace for more info)

The following warnings were emitted during compilation:

warning: In file included from /Users/psmit/kaldi-cxx/target/debug/build/kaldi-cxx-b3443cbce5d29980/out/autocxx-build-dir/cxx/gen0.cxx:1:
warning: In file included from /Users/psmit/Code/kaldi/tools/openfst/include/fst/arc.h:15:
warning: In file included from /Users/psmit/Code/kaldi/tools/openfst/include/fst/expectation-weight.h:24:
warning: In file included from /Users/psmit/Code/kaldi/tools/openfst/include/fst/pair-weight.h:18:
warning: In file included from /Users/psmit/Code/kaldi/tools/openfst/include/fst/weight.h:16:
warning: /Users/psmit/Code/kaldi/tools/openfst/include/fst/compat.h:87:23: warning: comparison of integers of different signs: 'int' and 'std::basic_string<char>::size_type' (aka 'unsigned long') [-Wsign-compare]
warning:     for (int i = 0; i < data.size(); ++i) {
warning:                     ~ ^ ~~~~~~~~~~~
warning: In file included from /Users/psmit/kaldi-cxx/target/debug/build/kaldi-cxx-b3443cbce5d29980/out/autocxx-build-dir/cxx/gen0.cxx:1:
warning: In file included from /Users/psmit/Code/kaldi/tools/openfst/include/fst/arc.h:15:
warning: In file included from /Users/psmit/Code/kaldi/tools/openfst/include/fst/expectation-weight.h:24:
warning: In file included from /Users/psmit/Code/kaldi/tools/openfst/include/fst/pair-weight.h:18:
warning: In file included from /Users/psmit/Code/kaldi/tools/openfst/include/fst/weight.h:19:
warning: /Users/psmit/Code/kaldi/tools/openfst/include/fst/util.h:125:62: warning: unused parameter 'v' [-Wunused-parameter]
warning:   return internal::ReadContainerType(strm, c, [](decltype(c) v, int n) {});
warning:                                                              ^
warning: /Users/psmit/Code/kaldi/tools/openfst/include/fst/util.h:125:69: warning: unused parameter 'n' [-Wunused-parameter]
warning:   return internal::ReadContainerType(strm, c, [](decltype(c) v, int n) {});
warning:                                                                     ^
warning: /Users/psmit/Code/kaldi/tools/openfst/include/fst/util.h:130:62: warning: unused parameter 'v' [-Wunused-parameter]
warning:   return internal::ReadContainerType(strm, c, [](decltype(c) v, int n) {});
warning:                                                              ^
warning: /Users/psmit/Code/kaldi/tools/openfst/include/fst/util.h:130:69: warning: unused parameter 'n' [-Wunused-parameter]
warning:   return internal::ReadContainerType(strm, c, [](decltype(c) v, int n) {});
warning:                                                                     ^
warning: /Users/psmit/Code/kaldi/tools/openfst/include/fst/util.h:135:62: warning: unused parameter 'v' [-Wunused-parameter]
warning:   return internal::ReadContainerType(strm, c, [](decltype(c) v, int n) {});
warning:                                                              ^
warning: /Users/psmit/Code/kaldi/tools/openfst/include/fst/util.h:135:69: warning: unused parameter 'n' [-Wunused-parameter]
warning:   return internal::ReadContainerType(strm, c, [](decltype(c) v, int n) {});
warning:                                                                     ^
warning: In file included from /Users/psmit/kaldi-cxx/target/debug/build/kaldi-cxx-b3443cbce5d29980/out/autocxx-build-dir/cxx/gen0.cxx:1:
warning: In file included from /Users/psmit/Code/kaldi/tools/openfst/include/fst/arc.h:15:
warning: In file included from /Users/psmit/Code/kaldi/tools/openfst/include/fst/expectation-weight.h:24:
warning: In file included from /Users/psmit/Code/kaldi/tools/openfst/include/fst/pair-weight.h:18:
warning: /Users/psmit/Code/kaldi/tools/openfst/include/fst/weight.h:232:20: warning: unused parameter 'w1' [-Wunused-parameter]
warning:   W2 operator()(W1 w1) const {
warning:                    ^
warning: In file included from /Users/psmit/kaldi-cxx/target/debug/build/kaldi-cxx-b3443cbce5d29980/out/autocxx-build-dir/cxx/gen0.cxx:1:
warning: In file included from /Users/psmit/Code/kaldi/tools/openfst/include/fst/arc.h:15:
warning: /Users/psmit/Code/kaldi/tools/openfst/include/fst/expectation-weight.h:78:37: warning: unused parameter 'delta' [-Wunused-parameter]
warning:   PairWeight<X1, X2> Quantize(float delta = kDelta) const {
warning:                                     ^
warning: /Users/psmit/Code/kaldi/tools/openfst/include/fst/expectation-weight.h:110:74: warning: unused parameter 'w1' [-Wunused-parameter]
warning: inline ExpectationWeight<X1, X2> Divide(const ExpectationWeight<X1, X2> &w1,
warning:                                                                          ^
warning: /Users/psmit/Code/kaldi/tools/openfst/include/fst/expectation-weight.h:111:74: warning: unused parameter 'w2' [-Wunused-parameter]
warning:                                         const ExpectationWeight<X1, X2> &w2,
warning:                                                                          ^
warning: /Users/psmit/Code/kaldi/tools/openfst/include/fst/expectation-weight.h:112:52: warning: unused parameter 'typ' [-Wunused-parameter]
warning:                                         DivideType typ = DIVIDE_ANY) {
warning:                                                    ^
warning: In file included from /Users/psmit/kaldi-cxx/target/debug/build/kaldi-cxx-b3443cbce5d29980/out/autocxx-build-dir/cxx/gen0.cxx:1:
warning: In file included from /Users/psmit/Code/kaldi/tools/openfst/include/fst/arc.h:16:
warning: /Users/psmit/Code/kaldi/tools/openfst/include/fst/float-weight.h:303:50: warning: unused parameter 'typ' [-Wunused-parameter]
warning:                                       DivideType typ = DIVIDE_ANY) {
warning:                                                  ^
warning: /Users/psmit/Code/kaldi/tools/openfst/include/fst/float-weight.h:538:45: warning: unused parameter 'typ' [-Wunused-parameter]
warning:                                  DivideType typ = DIVIDE_ANY) {
warning:                                             ^
warning: /Users/psmit/Code/kaldi/tools/openfst/include/fst/float-weight.h:709:48: warning: unused parameter 'typ' [-Wunused-parameter]
warning:                                     DivideType typ = DIVIDE_ANY) {
warning:                                                ^
warning: In file included from /Users/psmit/kaldi-cxx/target/debug/build/kaldi-cxx-b3443cbce5d29980/out/autocxx-build-dir/cxx/gen0.cxx:1:
warning: In file included from /Users/psmit/Code/kaldi/tools/openfst/include/fst/arc.h:17:
warning: /Users/psmit/Code/kaldi/tools/openfst/include/fst/lexicographic-weight.h:91:38: warning: unused parameter 'delta' [-Wunused-parameter]
warning:   LexicographicWeight Quantize(float delta = kDelta) const {
warning:                                      ^
warning: In file included from /Users/psmit/kaldi-cxx/target/debug/build/kaldi-cxx-b3443cbce5d29980/out/autocxx-build-dir/cxx/gen0.cxx:1:
warning: In file included from /Users/psmit/Code/kaldi/tools/openfst/include/fst/arc.h:20:
warning: /Users/psmit/Code/kaldi/tools/openfst/include/fst/signed-log-weight.h:157:48: warning: unused parameter 'typ' [-Wunused-parameter]
warning:                                     DivideType typ = DIVIDE_ANY) {
warning:                                                ^
warning: In file included from /Users/psmit/kaldi-cxx/target/debug/build/kaldi-cxx-b3443cbce5d29980/out/autocxx-build-dir/cxx/gen0.cxx:1:
warning: In file included from /Users/psmit/Code/kaldi/tools/openfst/include/fst/arc.h:21:
warning: /Users/psmit/Code/kaldi/tools/openfst/include/fst/sparse-power-weight.h:109:51: warning: unused parameter 'k' [-Wunused-parameter]
warning:   return SparsePowerWeightMap(w1, w2, [](const K &k, const W &v1, const W &v2) {
warning:                                                   ^
warning: /Users/psmit/Code/kaldi/tools/openfst/include/fst/sparse-power-weight.h:118:51: warning: unused parameter 'k' [-Wunused-parameter]
warning:   return SparsePowerWeightMap(w1, w2, [](const K &k, const W &v1, const W &v2) {
warning:                                                   ^
warning: /Users/psmit/Code/kaldi/tools/openfst/include/fst/sparse-power-weight.h:129:47: warning: unused parameter 'k' [-Wunused-parameter]
warning:                               [type](const K &k, const W &v1, const W &v2) {
warning:                                               ^
warning: /Users/psmit/Code/kaldi/tools/openfst/include/fst/sparse-power-weight.h:151:32: warning: unused parameter 'k' [-Wunused-parameter]
warning:       w1, w2, [delta](const K &k, const W &v1, const W &v2) {
warning:                                ^
warning: In file included from /Users/psmit/kaldi-cxx/target/debug/build/kaldi-cxx-b3443cbce5d29980/out/autocxx-build-dir/cxx/gen0.cxx:1:
warning: In file included from /Users/psmit/Code/kaldi/tools/openfst/include/fst/arc.h:22:
warning: /Users/psmit/Code/kaldi/tools/openfst/include/fst/string-weight.h:95:31: warning: unused parameter 'delta' [-Wunused-parameter]
warning:   StringWeight Quantize(float delta = kDelta) const { return *this; }
warning:                               ^
warning: /Users/psmit/Code/kaldi/tools/openfst/include/fst/string-weight.h:288:31: warning: unused parameter 'delta' [-Wunused-parameter]
warning:                         float delta = kDelta) {
warning:                               ^
warning: 23 warnings generated.

error: could not compile `kaldi-cxx` due to previous error
```

Running with higher log level shows the command that fails:
```
Caused by:
  process didn't exit successfully: `CARGO=/usr/local/Cellar/rust/1.57.0/bin/cargo CARGO_BIN_NAME=kaldi-cxx CARGO_CRATE_NAME=kaldi_cxx CARGO_MANIFEST_DIR=/Users/psmit/kaldi-cxx CARGO_PKG_AUTHORS='' CARGO_PKG_DESCRIPTION='' CARGO_PKG_HOMEPAGE='' CARGO_PKG_LICENSE='' CARGO_PKG_LICENSE_FILE='' CARGO_PKG_NAME=kaldi-cxx CARGO_PKG_REPOSITORY='' CARGO_PKG_VERSION=0.1.0 CARGO_PKG_VERSION_MAJOR=0 CARGO_PKG_VERSION_MINOR=1 CARGO_PKG_VERSION_PATCH=0 CARGO_PKG_VERSION_PRE='' CARGO_PRIMARY_PACKAGE=1 DYLD_FALLBACK_LIBRARY_PATH='/Users/psmit/kaldi-cxx/target/debug/deps:/usr/local/Cellar/rust/1.57.0/lib:/Users/psmit/lib:/usr/local/lib:/usr/lib' OUT_DIR=/Users/psmit/kaldi-cxx/target/debug/build/kaldi-cxx-b3443cbce5d29980/out rustc --crate-name kaldi_cxx --edition=2021 src/main.rs --error-format=json --json=diagnostic-rendered-ansi --crate-type bin --emit=dep-info,link -C embed-bitcode=no -C split-debuginfo=unpacked -C debuginfo=2 -C metadata=5d0a02be9fd0a6b5 -C extra-filename=-5d0a02be9fd0a6b5 --out-dir /Users/psmit/kaldi-cxx/target/debug/deps -C incremental=/Users/psmit/kaldi-cxx/target/debug/incremental -L dependency=/Users/psmit/kaldi-cxx/target/debug/deps --extern autocxx=/Users/psmit/kaldi-cxx/target/debug/deps/libautocxx-2773f7c2193b9e6a.rlib -L native=/Users/psmit/kaldi-cxx/target/debug/build/kaldi-cxx-b3443cbce5d29980/out -L /Users/psmit/Code/kaldi/tools/openfst/lib -l static=fst-demo -l c++ -l fst -L native=/Users/psmit/kaldi-cxx/target/debug/build/cxx-176f1ba771afaaee/out -L native=/Users/psmit/kaldi-cxx/target/debug/build/link-cplusplus-474ec68cdf5bd56e/out` (exit status: 1)
  ```
  
  The file I'm trying to include is from openfst 1.7.4 ( https://github.com/mjansche/openfst/blob/master/src/include/fst/arc.h )
  
  I'm expecting that I'm doing something very basic wrong, can someone point me in the right direction?

--- a/README.md
+++ b/README.md
@@ -41,11 +41,12 @@ You will need:
 
 The rest of this 'getting started' section assumes Cargo - if you're using something else, see the [`include_cpp`](https://docs.rs/autocxx/latest/autocxx/macro.include_cpp.html) documentation.
 
-First, add `autocxx` to your `dependencies` and `autocxx-build` to your `build-dependencies` in your `Cargo.toml`.
+First, add `autocxx` and `cxx` to your `dependencies` and `autocxx-build` to your `build-dependencies` in your `Cargo.toml`.
 
 ```toml
 [dependencies]
 autocxx = "0.15.0"
+cxx = "1.0"
 
 [build-dependencies]
 autocxx-build = "0.15.0"

